tests/form_autofill/test_updating_address.py

```diff
@@ -0,0 +1,64 @@
+import time
```

Comment:
Are you using `time` somewhere in here? I don't see it.

Reply:
I did, while creating the test, time.sleep, to help me with visualization. I just forget to remove it. I did another commit to remove the import.